py3status/modules/sysdata.py

```diff
@@ -126,25 +126,31 @@ def memory(self, unit='GiB'):
         # Otherwise, results are in gigabytes.
         return total_mem, used_mem, used_mem_p, unit
 
-    def cpuTemp(self, zone):
+    def cpuTemp(self, zone, unit):
         """
         Tries to determine CPU temperature using the 'sensors' command.
         Searches for the CPU temperature by looking for a value prefixed
         by either "CPU Temp" or "Core 0" - does not look for or average
         out temperatures of all codes if more than one.
         """
 
-        sensors = None
+        command = ['sensors']
+        if unit.lower() == 'F' or unit == u'°F':
```

Comment:
This is broken. 
```
unit = unit.lower()
if unit in [...]:
    ...
```

would be better.

Personally I'd just have unit as 'C', 'F', or 'K' without the degree but maybe that's just me